src-ui/app/edit-screen/components/mapping-pane/VariantDisplay.cpp

```diff
@@ -760,6 +763,7 @@ void VariantDisplay::showSRCMenu()
     };
     add(dsp::InterpolationTypes::Sinc, "Sinc");
     add(dsp::InterpolationTypes::Linear, "Linear");
+    add(dsp::InterpolationTypes::ZeroOrderHoldAA, "Zero-order Hold with antialiasing");
```

Comment:
I would shorten this to `Zero-Order Hold (Anti-Aliased)`. Note capitalization (I would also modify the capitalization of the ZOH mode below to match, while at it).